docker-compose.yml

```diff
@@ -1,34 +1,48 @@
 version: '3.1'
 
-services:
-    bracket-frontend:
-        image: ghcr.io/evroon/bracket-frontend
-        container_name: bracket-frontend
-        ports:
-            - "3000:3000"
-        environment:
-            NEXT_PUBLIC_API_BASE_URL: "http://localhost:8400"
-            NEXT_PUBLIC_HCAPTCHA_SITE_KEY: "10000000-ffff-ffff-ffff-000000000001"  # Public test key
-        restart: unless-stopped
+networks:
+  bracket_lan:
+    driver: bridge
```

Comment:
What is the reason to have a separate network in the development environment?
I can understand the purpose in production, but for development I don't see the reason.

Reply:
On some hosts the default network is fully isolated. Namely windows and wsl. Which means that by not having a network setup the containers cannot talk.